demos/rv-leaflet/src/ts/components/leaflet-example/leaflet-example.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component } from "@ribajs/core";
+
+import template from "./leaflet-example.component.html";
+
+export class LeafletExampleComponent extends Component {
+  public static tagName = "leaflet-example";
+
+  protected autobind = true;
+
+  static get observedAttributes() {
+    return [];
+  }
+
+  protected scope = {};
+
+  constructor(element?: HTMLElement) {
+    super(element);
+    console.debug("constructor", this);
```

Comment:
ist vermutlich noch von mir so übernommen aber kann mittlerweile zu `this.debug` geändert werden